data/json/items/resources/misc.json

```diff
@@ -282,5 +282,50 @@
     "material": [ "rubber", "steel" ],
     "symbol": "o",
     "color": "dark_gray"
+  },
+  {
+    "id": "hive_starter_immature",
```

Comment:
You don't need the two-step process here - ideally you rehome a found swarm directly, and they don't start pumping out workers without having found an appropriate home (that a sealed glass jar very much isn't). If the swarm works as a liquid you can just use that in the construction recipe.